spec/index.html

```diff
@@ -2383,12 +2381,155 @@ <h4>YamlLdErrorCode</h4>
           </section>
         </section>
 
+        <section>
+          <h2>Implementations</h2>
+
+          <p>
+            TODO: Implementations for Extended Internal Representation.
+          </p>
+        </section>
+
       </section>
 
       <section>
         <h2>Convert Extended YAML-LD to Basic YAML-LD and back</h2>
 
-        TODO
+        <p>
+          This approach is simpler than the Extended Internal Representation because it
+          does not require any changes to the internal structures of existing JSON-LD
+          libraries.
+        </p>
+
+        <p>
+          Instead, we implement two API functions:
+        </p>
+
+        <dl data-sort>
+          <dt><code>extended_to_basic(extended_document: YAML-LD) → YAML-LD</code></dt>
+          <dd>
+            <ul>
+              <li>Performs JSON-LD expansion to the given Extended YAML-LD document</li>
+              <li>Converts it to Basic YAML-LD form</li>
+            </ul>
+          </dd>
+
+          <dt><code>basic_to_extended(basic_document: YAML-LD) → YAML-LD</code></dt>
+          <dd>
+            <ul>
+              <li>Performs JSON-LD expansion to the given Basic YAML-LD document</li>
```

Comment:
I still don't understand what JSON-LD is doing in a YAML-LD-to-YAML-LD transformation.

Reply:
Commented above on that :white_check_mark: